packages/node/src/lib/notificationGroups/notificationGroups.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { Novu } from '../novu';
+import axios from 'axios';
+
+const mockConfig = {
+  apiKey: '1234',
+};
+
+jest.mock('axios');
+
+describe('test use of novus node package - NotificationGroups class', () => {
+  const mockedAxios = axios as jest.Mocked<typeof axios>;
+  let novu: Novu;
+
+  beforeEach(() => {
+    mockedAxios.create.mockReturnThis();
+    novu = new Novu(mockConfig.apiKey);
+  });
+
+  test('should get notification groups correctly', async () => {
+    mockedAxios.get.mockResolvedValue({});
+
+    await novu.notificationGroups.get();
+
+    expect(mockedAxios.get).toHaveBeenCalled();
+    expect(mockedAxios.get).toHaveBeenCalledWith('/notification-groups');
+  });
+
+  test('should update notification groups correctly', async () => {
```

Comment:
Shouldn't it be "should create ..."?

Reply:
Makes sense. I was not completely sure about the convention in the client. Updating to `create`. Thanks